test/integration/cms/toolbar_test.rb

```diff
@@ -8,9 +8,12 @@ def setup
   end
 
   test 'CMS toolbar rendering' do
+    cms_token = @provider.settings.cms_token!
+    expires_at = Time.now.utc + 1.minute.to_i
```

Comment:
a nitpick
```suggestion
    expires_at = Time.now + 1.minute.to_i
```

`#to_i` makes UTC anyway
```
[1] pry(main)> "#{Time.now.to_i} #{Time.now.utc.to_i}"
=> "1729874725 1729874725"
```

Reply:
I applied this, then reverted because Rubocop complains about `Time.now`